modules/test/playwright/utils/util.ts

```diff
@@ -3,6 +3,36 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
+import * as fs from 'fs';
+import * as path from 'path';
+
+const TMP_DIR = `tmp/${process.pid}`;
+
+export function createTempFile(name: string, content: string = ''): string {
+	fs.mkdirSync(TMP_DIR, {recursive: true});
+
+	const filePath = path.join(TMP_DIR, name);
+
+	if (fs.existsSync(filePath)) {
+		throw new Error(`Temporary file ${name} already exists`);
+	}
```

Comment:
What if multiple tests need to use this util? With this, it look to me like only the first one will be able to. To avoid this, we could allow overwrite, or provide a delete util.

Reply:
If two tests use the same temp file then I guess it should be named differently, otherwise they won't be able to run in parallel.

That's why I have provided a [WORM](https://en.wikipedia.org/wiki/Write_once_read_many) API here: `createTempFile` for write, `readTempFile` for read.

Reply:
Fair enough, but isn't more cleaner to delete temporary junk after each or all test? Is there some automated process that does this?

Also, is it viable to run any test in parallel? Would it require multiple DXP instances running?

Reply:
> isn't more cleaner to delete temporary junk after each or all tests? Is there some automated process that does this?

Technically, in the CI it doesn't matter because things are cleared after test runs. In local it will be deleted every time you do `git clean` or `ant all`.

But it's OK, I'll try to delete it when the test finishes.

Reply:
> is it viable to run any test in parallel? Would it require multiple DXP instances running?

Not yet and I'm not sure if we will ever be able for all tests because if the DXP instance is shared, it's difficult to guarantee that tests don't interact when system level settings are changed. 

But we should definitely try to write our tests with parallelization in mind just in case.

Also, fixtures like the FFs one will provide a way to inspect tests for their DXP state needs and may allow us to group them so that they can be parallelized by running those needing the same DXP state against the same DXP instance.

Reply:
> But it's OK, I'll try to delete it when the test finishes.

This is proving very difficult because the global setup and tearDown processes don't run once per worker, so there's no easy way to run a post-everything process.

I'll defer it to a future PR.

Reply:
I have finally implemented the temp files cleanupt using signal handlers from Node. Will push the changes soon.